### PR TITLE
Add CUDF 0.8 and 0.9

### PR DESCRIPTION
**Pull request by dra27**

### CODE REVIEW

**Comment by dra27** [comment]
Investigating a problem with the 0.9 package

**Comment by dra27** [comment]
Still looking into what's going on, but for some reason the change in hashtable sizes introduced in CUDF 0.8 breaks opam-solver (testsuite fails), so for now constraining that back to 0.7.

Issue with 0.9 is, slightly embarrassingly, because a DESTDIR change which I submitted for Cygwin doesn't work properly - fix is to add a forward slash in the opam file for now.

**Comment by avsm** [comment]
There appears to be a problem uninstalling the package as well...

```
#=== ERROR while installing cudf.0.7 ==========================================#
# opam-version 1.2.2
# os           linux
# command      make install DESTDIR=/home/travis/.opam/4.04.2 OCAMLLIBDIR=lib BINDIR=bin LIBDIR=lib INCDIR=include
# path         /home/travis/.opam/4.04.2/build/cudf.0.7
# compiler     4.04.2
# exit-code    2
# env-file     /home/travis/.opam/4.04.2/build/cudf.0.7/cudf-19855-4fc8eb.env
# stdout-file  /home/travis/.opam/4.04.2/build/cudf.0.7/cudf-19855-4fc8eb.out
# stderr-file  /home/travis/.opam/4.04.2/build/cudf.0.7/cudf-19855-4fc8eb.err
### stdout ###
# test -d /home/travis/.opam/4.04.2/lib || mkdir -p /home/travis/.opam/4.04.2/lib
# ocamlfind install -destdir /home/travis/.opam/4.04.2/lib -patch-version 0.7 cudf META _build/cudf.cma _build/cudf.cmxa _build/cudf.a _build/cudf_822_lexer.cmi _build/cudf_822_parser.cmi _build/cudf_c.cmi _build/cudf_checker.cmi _build/cudf_conf.cmi _build/cudf_parser.cmi _build/cudf_printer.cmi _build/cudf_type_lexer.cmi _build/cudf_type_parser.cmi _build/cudf_types.cmi _build/cudf_types_pp.cmi _build/cudf_822_parser.mli _build/cudf_checker.mli _build/cudf_conf.mli _build/cudf.mli _build/cudf_parser.mli _build/cudf_printer.mli _build/cudf_type_parser.mli _build/cudf_types.mli _build/cudf_types_pp.mli _build/cudf_822_lexer.cmx _build/cudf_822_parser.cmx _build/cudf_c.cmx _build/cudf_checker.cmx _build/cudf_conf.cmx _build/cudf_parser.cmx _build/cudf_printer.cmx _build/cudf_type_lexer.cmx _build/cudf_type_parser.cmx _build/cudf_types.cmx _build/cudf_types_pp.cmx _build/cudf_822_lexer.o _build/cudf_822_parser.o _build/cudf_checker.o _build/cudf_c.o _build/cudf_conf.o _build/cudf_parser.o _build/cudf_printer.o _build/cudf_type_lexer.o _build/cudf_type_parser.o _build/cudf_types.o _build/cudf_types_pp.o _build/cudf.o _build/cudf.cmx _build/cudf.cmi
### stderr ###
# ocamlfind: Package cudf is already installed
#  - (file /home/travis/.opam/4.04.2/lib/cudf/META already exists)
# make: *** [install] Error 2
```

**Comment by dra27** [comment]
Oops - hopefully fixed that now

**Comment by dra27** [comment]
Looks like the OSX test on Travis need a kick...

**Comment by dbuenzli** [comment]
I restarted the job.

**Comment by avsm** [comment]
Merging as osx breakage is infrastructure. thanks!

File: packages/cudf/cudf.0.9/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+authors: ["Roberto di Cosmo <roberto@dicosmo.org>" "Stefano Zacchiroli" "Pietro Abate"]
+homepage: "http://www.mancoosi.org/cudf/"
+bug-reports: "https://gforge.inria.fr/tracker/?atid=13811&group_id=4385&func=browse"
+dev-repo: "https://scm.gforge.inria.fr/anonscm/git/cudf/cudf.git"
+build: [make "all" "opt"]
+remove: [
+ [make "uninstall" "DESTDIR=%{prefix}%/" "OCAMLLIBDIR=lib" "BINDIR=bin" "LIBDIR=lib" "INCDIR=include"]
+]
+depends: [
+  "conf-perl" {build}
```
Comment (avsm) [inline packages/cudf/cudf.0.9/opam]: ahh I see that the camlp4 dependency has finally disappeared :-)

**Pull request merged**